packages/expo-apple-authentication/src/AppleAuthentication.types.ts

```diff
@@ -185,6 +185,22 @@ export type AppleAuthenticationFullName = {
   nickname: string | null;
 };
 
+// @needsAudit @docsMissing
+/**
+ * An enum whose values specify the style for formatting a full name when calling `AppleAuthentication.formatFullName()`.
+ *
+ * @see [Apple
+ * Documentation](https://developer.apple.com/documentation/foundation/personnamecomponentsformatter)
+ * for more details.
+ */
+export enum AppleAuthenticationFullNameFormatStyle {
+  DEFAULT = 0,
+  SHORT = 1,
+  MEDIUM = 2,
+  LONG = 3,
+  ABBREVIATED = 4,
+}
```

Comment:
We can use a string union here
```suggestion
export type AppleAuthenticationFullNameFormatStyle  =
  "default" |
  "short" |
  "medium" |
  "long" |
  "abbreviated"
}
```

Reply:
Out of curiosity, why use a string union? I originally made it an enum since I noticed this specific library used enums heavily